riptable/rt_datetime.py

```diff
@@ -837,7 +837,7 @@ def strftime(self, format, dtype='O'):
 
         '''
         if isinstance(self, np.ndarray):
-            return np.asarray([dt.utcfromtimestamp(timestamp).strftime(format) for timestamp in self._fa* SECONDS_PER_DAY], dtype=dtype)
+            return np.asarray([dt.utcfromtimestamp(timestamp).strftime(format) for timestamp in self._fa.astype(np.int64) * SECONDS_PER_DAY], dtype=dtype)
         else:
             return dt.strftime(dt.utcfromtimestamp(self * SECONDS_PER_DAY), format)

```

Comment:
considering `DateBase` derives from `FastArray` and so is always an `np.ndarray`, I'm not sure how this statement could be reached, so not changing it.